src/python/system/environment.py

```diff
@@ -1066,6 +1066,14 @@ def is_android_emulator(plt=None):
   return 'ANDROID_EMULATOR' == (plt or platform())
 
 
+def is_android_kernel(plt=None):
+  """Return true if we are running android kernel jobs."""
+  if not plt:
+    queue_override = get_value('QUEUE_OVERRIDE')
+    plt = queue_override if queue_override else platform()
+  return 'ANDROID_KERNEL' in plt
```

Comment:
Just return `'ANDROID_KERNEL' in platform_group()`. `platform_group` does the same thing you are doing here.